### DISCUSSION TITLE
Request replace hard coded route('login')  to login redirect from config settings

### DISCUSSION THREAD

**Discussion by ay4t** [codeigniter4/shield#1123] [General]
If I access my application and I no longer have a login session, Shield will always redirect to the route with name login. Whereas I use a custom URL with the name auth.login or <domain>/auth/login

I think it is still possible to change the redirect according to the redirect config

![2024-05-20_21-49](https://github.com/codeigniter4/shield/assets/87152520/3d22019d-3ed1-4473-bd97-a69036b41d42)
![2024-05-20_21-53](https://github.com/codeigniter4/shield/assets/87152520/15b3c26b-7f38-4d01-98fb-279466efd200)

**Comment by warcooft** [comment]
First, open config file in `app/Config/Auth.php`
and change logout redirect value as you want 

```php
 public array $redirects = [
        'register'          => '/',
        'login'             => '/admin',
        'logout'            => '/your-custom-url-here',
        // ...
    ];
``` 

Make sure to use the default logout URL route provided by Shield, for example in your case: [binary.localhost/logout](url)

if you use a custom logout route, add a redirect page to:
```php
$url = config('Auth')->logoutRedirect();

```
